scripts/zulip_emoji_merge_delegate.py

```diff
@@ -23,15 +23,34 @@
     site=ZULIP_SITE
 )
 
-# Fetch the last 200 messages
-response = client.get_messages({
+# Fetch the messages containing the PR number from the public channels.
+# There does not seem to be a way to search simultaneously public and private channels.
+public_response = client.get_messages({
     "anchor": "newest",
-    "num_before": 200,
+    "num_before": 5000,
     "num_after": 0,
-    "narrow": [{"operator": "channel", "operand": "PR reviews"}],
+    "narrow": [
+        {"operator": "channels", "operand": "public"},
+        {"operator": "search", "operand": f'{PR_NUMBER}'},
```

Comment:
Does it make sense to search for #NNNN rather than NNNN?

Reply:
I am not sure how Zulip's search matching works, but I agree it might be worth seeing if we can avoid easy false positives (for example will the bot tag lean4 PR links, etc.)

Reply:
I tested it and it seems that adding the `#` works: I pushed this and will merge once it is green!

Reply:
By the way, the PR number is a first screening.  After that, there is a second filter that looks for the actual URL of the PR.